src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookConnector.java

```diff
@@ -41,22 +45,41 @@ public WebhookConnector(String projectKey, String webhookUrl, String webhookSecr
 			this.rootTypes = rootTypes;
 		}
 		this.elementIds = new ArrayList<Long>();
+
+		this.receiver  = "Other";
+		if (this.url != null){
+			if(url.matches("https://hooks.slack.com(\\S*)")) {
+			this.receiver = "Slack";}
+		}
+
+
 	}
 
 	public WebhookConnector(String projectKey) {
-		this(projectKey, ConfigPersistenceManager.getWebhookUrl(projectKey),
+		this(projectKey,
+				ConfigPersistenceManager.getWebhookUrl(projectKey),
 				ConfigPersistenceManager.getWebhookSecret(projectKey),
 				ConfigPersistenceManager.getEnabledWebhookTypes(projectKey));
 	}
 
 	public boolean sendElementChanges(KnowledgeElement changedElement) {
+		//System.out.println("sendElementChanges. Receiver: "+this.receiver);
 		boolean isSubmitted = false;
 		if (!checkIfDataIsValid(changedElement)) {
 			return isSubmitted;
 		}
-		List<KnowledgeElement> rootElements = getWebhookRootElements(changedElement);
-		isSubmitted = postKnowledgeTrees(rootElements);
-		return isSubmitted;
+		if(this.receiver == "Other"){
+			List<KnowledgeElement> rootElements = getWebhookRootElements(changedElement);
+			isSubmitted = postKnowledgeTrees(rootElements);
+			return isSubmitted;
+	  }
+		if(this.receiver == "Slack"){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)